### ISSUE TITLE
Occasional screen tearing

### ISSUE THREAD

**Issue by proxyglitchcat** [xerpi/vita-udcd-uvc#87]
I'm currently on the latest release, and when I play specific games (such as Reel Fishing: Master's Challenge), I encounter horrifying levels of screen tearing very often that is not on the console itself. Attached is a clip from Twitch, the full stream shows this screen tearing more. https://clips.twitch.tv/BusyTangibleDogeBIRB-NFS9PiGwRJtPqD03

**Comment by xerpi** [comment]
Yeah that's right... the plugin needs a big code refactor to be improved in many aspects.
Regarding tearing, probably `ksceDisplayRegisterFrameBufCallback` should be used instead of https://github.com/xerpi/vita-udcd-uvc/blob/master/src/main.c#L780.
